src/components/Navigation/NavigationBar.tsx

```diff
@@ -0,0 +1,56 @@
+import Navbar from 'react-bootstrap/Navbar';
+import Container from 'react-bootstrap/Container';
+
+import './NavigationBar.scss';
+import logo from '../../resources/images/GenericSoft_Logo_White.svg';
+
+import DropDown from '../DropDown/DropDown';
+import UserImage from '../UserImage/UserImage';
+
+import { useAppSelector } from '../../redux/store';
+
+function NavigationBar() {
```

Comment:
Please, change it to arrow function. Also, rename the Navigation folder to NavigationBar or rename the .tsx NavigationBar file to Navigation in order to be consistent.